lib/index.ts

```diff
@@ -60,6 +61,45 @@ formula.AGGREGATE = <T>(list: T[], func: (item: T) => Iterable<T>): T[] => {
 	);
 };
 
+formula.NEEDS_ALL = (...statuses: NEEDS_STATUS[]) => {
+	let needsAllStatus = NEEDS_STATUS.MERGEABLE;
```

Comment:
```suggestion
	let result = NEEDS_STATUS.MERGEABLE;
```
too much "needs" is confusing me 😅